qt/python/mantidqt/utility/gui_test.py

```diff
@@ -0,0 +1,110 @@
+import types
```

Comment:
Similarly here (and all of the stuff in `utility`). As we now have the `mantidqt.utils` package can we move this to `mantidqt.utils.testing`?